.github/workflows/ci_l2_proving.yaml

```diff
@@ -0,0 +1,63 @@
+name: L2 (SP1 Backend)
```

Comment:
this doesn't run on a pr, so it's not `ci_`. I can rename all `ci_` to `pr_` to make it clearer.

Reply:
https://github.com/lambdaclass/ethrex/pull/2395